Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -27,38 +27,37 @@ public class DependencyCollectorDiagnosticListener : IObserver<DiagnosticListene
         /// <returns>
         /// An IDisposable that can be disposed to disable the DependencyCollectorDiagnosticListener.
         /// </returns>
-        public static IDisposable Enable(TelemetryConfiguration configuration = null)
+        public static IDisposable Enable(TelemetryConfiguration configuration = null, IEnumerable<string> excludedDomains = null)
```

Comment:
@SergeyKanzhelev Aren't we going to have to call this differently from the .NET Core SDK to provide the list of domains to exclude because we don't have the config file that the web SDK module has access to?